libc/utils/docgen/docgen.py

```diff
@@ -73,10 +172,78 @@ def print_header(header: str, api: Dict):
 
   * - Function
     - Implemented
-    - Standard"""
+    - C Standard
+    - POSIX Standard"""
```

Comment:
Hmm...the current wording makes me think "what C standard or POSIX standard was this first defined in" which is how the man pages project documents these, but NOT what we're doing for docgen.

Let's discuss (in this comment thread, before you post changes to wording) some different ways we can make that clearer.

Perhaps:
- C23 Standard Section
- POSIX Docs

Would make our intent clearer? Got any other ideas?